app/about_us/OurCrew.js

```diff
@@ -0,0 +1,68 @@
+import styles from "./ourCrew.module.css";
+const OurCrew = () => {
+  // TASK - React 1 week 1
+  // Create the "Our Crew section"
+  // Use the descriptions provided in /app/about_us/README.md
+  // Use the pictures from /public/crew
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <div>
+      <p>
+        Our crew is the heart and soul of Galactica. We are a diverse team of
+        seasoned space explorers, engineers, and visionaries who are united by a
+        common goal: to make space travel accessible and exciting for all.
+      </p>
+      <div className={styles.imageContainer}>
+        <div className={styles.imageItem}>
+          <img
+            src="/crew/image-anousheh-ansari.png"
+            alt="anousheh-ansari_image"
+          ></img>
+          <p className={styles.imageDesc}>
+            Captain Sarah Vega: A former NASA astronaut with over 15 years of
+            experience, Captain Vega leads our missions with unparalleled
+            expertise and a passion for space exploration.
+          </p>
+        </div>
+        <div className={styles.imageItem}>
+          <img
+            src="/crew/image-douglas-hurley.png"
+            alt="image-douglas-hurley"
```

Comment:
Oh, Thanks for the links. I thought the alt attribute is used to display the description of the image when the image fails to load. So that the user can understand what the image represents.